index.html

```diff
@@ -148,5 +148,31 @@ <h2>About me</h2>
         </ul>
     </ul>
   </section>
+  <section class="contact-section">
+    <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    <form class="contact-form" action="https://formspree.io/f/{form_id}" method="post">
+      <ul>
+        <li>
+          <input type="text"  name="name" placeholder="Full name" maxlength="30" required/>
+        </li>
+        <li>
+          <input type="email" name="email" placeholder="Email address"  required/>
+        </li>
+        <li>
+          <textarea id="msg" name="user_message" maxlength="500" placeholder="Write me something..." required></textarea>
```

Comment:
- [ ] Please ensure to give this input element padding-left and padding-top according to the design. 👍